src/App.jsx

```diff
@@ -1,12 +1,37 @@
 import { useState } from "react";
+import { Statistics } from "./Statistics";
+import { Anecdotes } from "./Anecdotes";
 
 const App = () => {
-  // save each button click in its own state
+  const anecdotes = [
+    "If it hurts, do it more often.",
+    "Adding manpower to a late software project makes it later!",
+    "The first 90 percent of the code accounts for the first 90 percent of the development time...The remaining 10 percent of the code accounts for the other 90 percent of the development time.",
+    "Any fool can write code that a computer can understand. Good programmers write code that humans can understand.",
+    "Premature optimization is the root of all evil.",
+    "Debugging is twice as hard as writing the code in the first place. Therefore, if you write the code as cleverly as possible, you are, by definition, not smart enough to debug it.",
+    "Programming without an extremely heavy use of console.log is same as if a doctor would refuse to use x-rays or blood tests when diagnosing patients.",
+    "The only way to go fast, is to go well.",
+  ];
+
   const [good, setGood] = useState(0);
   const [neutral, setNeutral] = useState(0);
   const [bad, setBad] = useState(0);
 
-  return <div>code here</div>;
+  const handleGood = () => setGood(good + 1);
```

Comment:
Be careful with this setter implementation. In this case it's better to use a callback to retrieve previous value and update it.

FIX: fix setter and extract it to a handler

```js
const handleGoodIncrease = () => {
  setGood((prevGoodValue) => {
    return prevGoodValue + 1
  })
}

//...

<button onClick={handleGoodIncrease}>good</button>
```